components/MarkdownEditor/InternalEditor.vue

```diff
@@ -4,7 +4,7 @@
       class="mx-auto flex flex-wrap pb-1 gap-2 *:flex *:gap-1"
       role="menubar"
     >
-      <div role="group">
+      <div role="group" v-if="! raw">
```

Comment:
Can you move all the `<div>` in a template with a single `v-if` ?

Reply:
Done in dfd8e2a